.buildkite/commands/swiftlint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -u
+
+echo "--- :swift: Running SwiftLint"
+
+# Using --strict to fail the build and increase the likelyhood devs will notice the violations
+swiftlint_output=$(swiftlint lint --quiet --strict --reporter csv)
```

Comment:
cc @iangmaia what do you think of running in strict mode to make the violations more visible (failed builds don't go unnoticed)

Reply:
I think it's a good idea! I've actually created a separate PR focusing on this and a couple of other SwiftLint improvements for PocketCasts iOS with that change: https://github.com/Automattic/pocket-casts-ios/pull/1361

I'm actually also [tweaking the code](https://github.com/Automattic/pocket-casts-ios/pull/1326/commits/475e4e97936e4b7b79996d0687e45dc55fb4b8ec) a bit. Instead of using the `csv` reporter, I've changed it to `relative-path`, which shows a more sensible filepath but also has the advantage of not potentially breaking things if we have a violation description with a comma (it's something @AliSoftware brought to my attention in the PCiOS review).